src/puppetlabs/ezbake/core.clj

```diff
@@ -418,6 +421,11 @@ Additional uberjar dependencies:
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; Ephemeral Git Repo functions
 
+(defn get-git-sha
+  []
+  (clojure.string/trim-newline (:out (exec/exec "git" "rev-parse" "HEAD")))
```

Comment:
Looks like we already have `clojure.string` imported as `str`, so this can just be `str/trim-newline`